app/assets/stylesheets/hyrax/_footer.scss

```diff
@@ -22,6 +22,11 @@ body {
   bottom: 0;
   height: $footer-height;
   width: 100%;
+
+  a,
+  &.navbar-inverse .navbar-link {
```

Comment:
I see, you're changing the customization defaults below.  How about I add a customizable footer link color into the appearance menu instead?